moveit_ros/move_group/src/default_capabilities/execute_trajectory_action_capability.cpp

```diff
@@ -106,7 +106,7 @@ void MoveGroupExecuteTrajectoryAction::executePath(const std::shared_ptr<ExecTra
 {
   RCLCPP_INFO(getLogger(), "Execution request received");
 
-  context_->trajectory_execution_manager_->clear();
+  context_->trajectory_execution_manager_->stopExecution(true);
```

Comment:
To maintain the old behaviour of rejecting the execution request, you should remove the call completely.
```suggestion
```
https://github.com/moveit/moveit2/blob/932c19217ce0258a0213b89270e490807501bc8d/moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp#L386-L390